app/api/services/brief_overview.py

```diff
@@ -63,26 +25,49 @@ def brief_contains_any_optional_fields(self, brief, optional_fields):
         return False
 
     def is_brief_section_complete(self, brief, section):
+        from ...brief_utils import determine_required_fields
         if section['required']:
-            return self.brief_contains_all_required_fields(brief, section['required'])
+            required_fields = determine_required_fields(brief, section, enforce_required=False)
+            return self.brief_contains_all_required_fields(brief, required_fields)
         else:
             return self.brief_contains_any_optional_fields(brief, section['optional'])
 
     def get_publish_links(self, brief):
         links = []
         draft_brief = brief.status == 'draft'
-
+        brief_builder_paths = {
+            'How long your brief will be open': '{path}/edit/how-long-your-brief-will-be-open/requirementsLength',
+            'Description of work': '{path}/description-of-work',
+            'Location': '{path}/edit/location/location',
+            'Question and answer session details': '{path}/question-and-answer-session-details',
+            'Role': '{path}/edit/role/title',
+            'Shortlist and evaluation process': '{path}/shortlist-and-evaluation-process',
+            'Title': '{path}/edit/title/title',
+            'Who can respond': '{path}/edit/who-can-respond/specifySeller',
+            'Who can respond|training': '{path}/edit/who-can-respond%7Ctraining/specifySellerTraining',
```

Comment:
As discussed, check to see if this has been removed from the designs.

Reply:
removed